### ISSUE TITLE
FGType example in docs doesn't work

### ISSUE THREAD

**Issue by richardjgowers** [lukasturcani/stk#93]
https://stk.readthedocs.io/en/latest/stk.molecular.functional_groups.html#stk.molecular.functional_groups.FGType.get_functional_groups

The example listed here doesn't work, I get `TypeError: __init__() missing 1 required positional argument: 'name'`

**Comment by lukasturcani** [comment]
Yep, thanks for this. It should be

```python

amine = stk.FGType(
    func_group_smarts='[N]([H])[H]',
    bonder_smarts=['[$([N]([H])[H])]'],
    deleter_smarts=['[$([H][N][H])]']*2,
    name='amine',
)

```

**Comment by lukasturcani** [comment]
Fixed in #95
